### ISSUE TITLE
Parse error involving variables within variable definitions

### ISSUE THREAD

**Issue by elydpg** [alda-lang/alda-core#64]
Consider the following code:

```
soprano = >c< b >c<
alto = g g g
tenor = e d e
bass = c <b> c
piano: V1: soprano V2: alto V3: tenor V4: bass
```
Plays normally
However,

```
soprano = >c< b >c<
alto = g g g
tenor = e d e
bass = c <b> c

melody = V1: soprano V2: alto V3: tenor V4: bass

piano: melody
```
Throws the error `[27713] ERROR Unrecognized event: :name` (this happens regardless of whether or not the variable `melody` is actually used or not).
However,

```
soprano = >c< b >c<
alto = g g g
tenor = e d e
bass = c <b> c

melody = [V1: soprano V2: alto V3: tenor V4: bass]

piano: melody
```
Plays normally.

**Comment by daveyarwood** [comment]
Another nice catch @elyisgreat -- I've found the cause* and am working on a fix.

*Don't ask, it's a really weird edge case!

**Comment by daveyarwood** [comment]
...OK, I'll explain :)

Voices ended up actually being a red herring. It turns out that there's a bug in the parser that has to do with ending a variable definition with another variable reference.

This works because the `foo =` line ends with `f` (not a variable):

```
bar = c d e
foo = bar f

piano: foo
```

But this fails with `Unrecognized event: :name` because the reference to `bar` isn't getting disambiguated from a "name" token into a "get variable" token.

```
bar = c d e
foo = bar

piano: foo
```

Should have a fix out shortly!

**Comment by daveyarwood** [comment]
Fixed in [1.0.0-rc80](https://github.com/alda-lang/alda/releases/tag/1.0.0-rc80).
